osx/reset.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+#  This will remove the following.
+#    1. Unset environment variable: KALITE_PYTHON
+#    2. remove the .plist file, need admin
+#    3. delete the symlinked /usr/bin/kalite command, need admin
+
+KALITE_PLIST="org.learningequality.kalite.plist"
+TEMP="/tmp"
+LIBRARY="/Library/LaunchAgents"
+KALITE_EXECUTABLE_PATH="$(which kalite)"
+
+
+# Delete the .plist file.
+if [ -f "$TEMP/$KALITE_PLIST" ]; then
+    echo "Removing $TEMP/$KALITE_PLIST "
+    sudo rm "$TEMP/$KALITE_PLIST"
+else
+    echo "$KALITE_PLIST not found. "
+fi
+
+if [ -f "$LIBRARY/$KALITE_PLIST" ]; then
+    echo "Removing $LIBRARY/$KALITE_PLIST "
+    sudo rm "$LIBRARY/$KALITE_PLIST"
+else
+    echo "$LIBRARY/$KALITE_PLIST not found."
```

Comment:
Create a variable to hold these four (4) instances of the same string concatenation: `$LIBRARY/$KALITE_PLIST` and use that instead.